### PR TITLE
44 create api review issue template

### LINKED ISSUE #44: Create an api-review issue template for use by release management
**Problem description**

Create an api-review issue template for use by release management to coordinate API reviews by the team. It shall list the items to be reviewed and allow to link to Sub Project issues / PRs that need to be reviewed.  

**Expected action**

Create a new template in the RM project

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by tanjadegroot**
#### What type of PR is this?

Add one of the following kinds:
* enhancement/feature
* subproject management

#### What this PR does / why we need it:
Issue template for coordinating API release PR reviews by the Release Mgmt team

#### Which issue(s) this PR fixes:

Fixes #44 

#### Changelog input

```
 release-note

```

#### Additional documentation 

This section can be blank.

```
docs

```

### CODE REVIEW

**Comment by tanjadegroot** [comment]
@hdamker please check if this issue template looks OK. 
Given current content, I do not see any need to update the config.yaml file (unless it should be tailored more to the RM working group)

File: .github/ISSUE_TEMPLATE/issue_review_template [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+**Expected actions**
+To review, follow the below list. 
+For any item, when done, put a comment in this issue.
+
+* yaml (check version in info & servers objects)  
+* test file(s) availability
+* changelog updated
+* readme updated (enforce correct release number / API version naming) 
+* API readiness checklist
```
Comment (hdamker) [inline .github/ISSUE_TEMPLATE/issue_review_template:21]: How about allowing to check-off the points also - GitHub will show then a nice progress indicator within the issue overview:

```suggestion
[ ] API Definition files (YAML) (check version in info & servers objects)  
[ ] test file(s) availability
[ ] changelog updated
[ ] readme updated (enforce correct release number / API version naming) 
[ ] API readiness checklist(s)
```

**Review by hdamker — DISMISSED** [review]
Looks good to me - just a suggestion

**Comment by hdamker** [comment]
> Given current content, I do not see any need to update the config.yaml file (unless it should be tailored more to the RM working group)

Correct, let's do the tailoring within a separate issue.

**Comment by hdamker** [comment]
@tanjadegroot Let me know if I should merge it - I don't think we need a further review for that.

**Comment by tanjadegroot** [comment]
@hdamker - yes, greate suggestions and please merge

**Pull request merged**